account_payment_return_import/tests/test_import_payment_return.py

```diff
@@ -12,6 +12,7 @@ def setUp(self):
         self.company = self.env.ref('base.main_company')
         self.acc_number = 'NL77ABNA0574908765'
         self.acc_bank = self.env['res.partner.bank'].create({
+            'partner_id': self.env.ref('base.res_partner_1').id,
```

Comment:
Can you change with this:
`'partner_id': self.company.partner_id.id`
for it hasn't demo data dependences

Should not the company be?:
`'company_id': self.company.id`

Reply:
Done

Reply:
Thanks